css-env-1/explainers/env-preferred-text-scale.md

```diff
@@ -383,7 +383,11 @@ If an author didn’t want headings – for example – to scale up at the same
 
 By default, [Android Webview scales the text size by the system font scale factor](https://crsrc.org/c/android_webview/java/src/org/chromium/android_webview/AwSettings.java;l=382-385;drc=fbd39d07c1deaf2ec4a8ea32260503369d10bdbb). App developers may override this default by invoking [setTextZoom()](<https://developer.android.com/reference/android/webkit/WebSettings#setTextZoom(int)>).
 
+**Update 2025-Jul-08:** The interaction between WebView and env() is in flux. See https://groups.google.com/a/chromium.org/g/blink-dev/c/bZuQAcwcEig/m/6F5w5S97AQAJ
```

Comment:
I recommend giving the links text just so that they're more accessible.

```suggestion
**Update 2025-Jul-08:** The interaction between WebView and env() is in flux. See the [discussion on blink-dev.](https://groups.google.com/a/chromium.org/g/blink-dev/c/bZuQAcwcEig/m/6F5w5S97AQAJ)
```